pages/common/pest.md

```diff
@@ -0,0 +1,24 @@
+# pest
+
+> A PHP testing framework with a focus on simplicity.
+> More information: <https://pestphp.com>.
+
+- Initialise a standard Pest configuration in the current directory:
```

Comment:
We have pages that say `local path`, I think we need to uniform the expression for `current directory`/`local path`:
```suggestion
- Initialise a standard Pest configuration in the local path:
```

Reply:
Hmm, personally I feel that current directory makes more sense than local path. 🤔 🤷🏻

Reply:
I agree. `current directory` makes more sense.